tutorials/social-recovery/blockchain_app/srs_module/index.js

```diff
@@ -1,3 +1,58 @@
-const SRSModule = require('./srs_module');
+const { BaseModule, codec } = require('lisk-sdk');
+const  { CreateRecoveryAsset, CREATE_RECOVERY_ASSET_ID } = require('./assets/create_recovery');
+const { InitiateRecoveryAsset, INITIATE_RECOVERY_ASSET_ID } = require('./assets/initiate_recovery');
+const VouchRecovery = require('./assets/vouch_recovery');
+const ClaimRecovery = require('./assets/claim_recovery');
+const CloseRecovery = require('./assets/close_recovery');
+const { RemoveRecoveryAsset, REMOVE_RECOVERY_ASSET_ID } = require('./assets/remove_recovery');
+const { SRSAccountSchema, createRecoverySchema, initiateRecoverySchema } = require('./schemas');
+
+// Extend from the base module to implement a custom module
+class SRSModule extends BaseModule {
+  name = 'srs';
+  id = 1026;
+  accountSchema = SRSAccountSchema;
+
+  transactionAssets = [
+    new CreateRecoveryAsset(),
+    new InitiateRecoveryAsset(),
+    new VouchRecovery(),
+    new ClaimRecovery(),
+    new CloseRecovery(),
+    new RemoveRecoveryAsset(),
+  ];
+
+  events = ['createdConfig','removedConfig','initiatedRecovery'];
```

Comment:
```suggestion
  events = ['configCreated','configRemoved','initiatedRecovery'];
```
maybe this sounds better but not a strong opinion on this